src/mscorlib/shared/System/IO/MemoryStream.cs

```diff
@@ -235,15 +227,13 @@ internal byte[] InternalGetBuffer()
         // PERF: True cursor position, we don't need _origin for direct access
         internal int InternalGetPosition()
         {
-            if (!_isOpen) throw Error.GetStreamIsClosed();
```

Comment:
It looks like InternalGetPosition is used by BinaryReader.  Does removing this have any negative impact there?

Reply:
The one place where it is called is immediately followed by a call to InternalEmulateRead that verifies that stream is not closed.

Reply:
Cool